src/test/java/org/folio/helper/PurchaseOrderHelperTest.java

```diff
@@ -178,7 +158,52 @@ void testPostOpenCompositeOrder() throws Exception {
       .when(encumbranceService).updateEncumbrancesOrderStatusAndReleaseIfClosed(any(CompositePurchaseOrder.class),
         eq(requestContext));
 
-    Future<CompositePurchaseOrder> future = purchaseOrderHelper.postCompositeOrder(compositePo, requestContext);
+    // When
```

Comment:
comments is confusing the code above there are many when. It is better to remove

Reply:
I don't understand, structuring the tests with Given/When/Then is useful to clarify and to read the tests (another TJ developer started the practice). Are you suggesting we remove all the Given/When/Then comments ?

Reply:
yes, It is better to remove comments. From my prespective it is clear structure of test. Usually this strucutre is confusing

```
//Given
object = new

when...
when..

//When
operation

//Then
```

Reply:
I don't see any issues with that and we don't have any convention on using or not using //Given //When //Then so we can remain as is without changes

Reply:
OK, I will keep that for now, but we should probably have a discussion and vote. I understand Aziz' concern.